src/main/java/edu/cornell/kfs/tax/dataaccess/impl/Transaction1099Summary.java

```diff
@@ -198,9 +225,117 @@
         
         this.taxBoxIncludedDvCheckStubTextMaps = Collections.unmodifiableMap(tempCheckStubTextMaps);
         this.taxBoxDvCheckStubTextsAreWhitelists = Collections.unmodifiableMap(tempWhitelistFlagsMap);
+        this.taxBoxMinimumReportingAmounts = parseReportingThresholdsForTaxBoxes(parameterService, boxNumberMappings);
+        this.defaultTaxBoxMinimumReportingAmount = computeDefaultTaxBoxMinimumReportingAmount();
+        this.filerAddressFields = parseFilerAddressFields(parameterService);
+    }
+
+    private Map<Tax1099FilerAddressField, String> parseFilerAddressFields(ParameterService parameterService) {

```

Comment:
I wonder if a better name would be buildFilerAddressFieldsMapFromParameter?

Reply:
Ok, I'll rework this method to give it a better name, like the one you suggested.